libc-test/build.rs

```diff
@@ -145,11 +145,41 @@ fn main() {
     // Avoid unnecessary re-building.
     println!("cargo:rerun-if-changed=build.rs");
 
+    if std::fs::exists("../target/src-hotfix").unwrap() {
+        std::fs::remove_dir_all("../target/src-hotfix").unwrap();
+    }
+
+    // ctest2 cannot parse `crate::` in paths, so replace them with `::`
```

Comment:
Could you make this `// FIXME(ctest): ...` so it's greppable with the other ctest problems?

Reply:
Done